### PR TITLE
(Fix) Rebase issues and top contributors count increased to 6 developers

### PR DESCRIPTION
**Pull request by AVRahul**
Signed-off-by: AVRahul <a.v.rahul11@gmail.com>

- Fix some of the rebasing issues in the previous PR.
- Increased top contributors to 6, also added the link for the `dev.stats` of top contributors.
 
Have closed the other PR in order to avoid conflicts. https://github.com/openebs/website/pull/236

**Note:** Also, have added a Column for new contributors and made responsive but commented on the code till the action data is been fetched.

### CODE REVIEW

**Review by IsAmrish — APPROVED** [review]

**Review by rakeshPRaghu — APPROVED** [review]
LGTM

File: website/src/components/Footer/index.tsx [hunk 1]
```diff
@@ -146,61 +134,102 @@ const Footer: React.FC = () => {
...
-              >
-                {contributor}
-              </Link>
-            ))}
-          </Typography>
-        </>
+          </Link>
+        </Typography>
+        <Typography className={classes.columnListWrapper}>
+          {topContributors?.slice(0, 6).map((contributor: any) => (
```
Comment (Pallavi-PH) [inline website/src/components/Footer/index.tsx:147]:
```suggestion
          {topContributors?.slice(0, 6).map((contributor: string) => (
```

File: website/src/components/Footer/index.tsx [hunk 2]
```diff
@@ -146,61 +134,102 @@ const Footer: React.FC = () => {
...
+  //       <Typography variant="h6" className={classes.columnTitle}>
+  //         <Link
+  //           href={EXTERNAL_LINKS.TOP_GITHUB_CONTRIBUTORS_URL}
+  //           target="_blank"
+  //         >
+  //           {t('footer.newContributors')}
+  //         </Link>
+  //       </Typography>
+  //       <Typography className={classes.columnListWrapper}>
+  //         {topContributors?.slice(0, 6).map((contributor: any) => (
```
Comment (Pallavi-PH) [inline website/src/components/Footer/index.tsx:176]:
```suggestion
  //         {topContributors?.slice(0, 6).map((contributor: string) => (
```

File: website/src/pages/Home/index.tsx [hunk 3]
```diff
@@ -43,590 +36,405 @@ const Home: React.FC = () => {
...
-      text: 'helm install openebs --namespace openebs openebs/openebs --create-namespace',
-      status: 'Copied to clipboard',
-    });
-    setTimeout(() => {
+    const copyToClipboard = () => {
+      navigator.clipboard.writeText(copyCommand.text);
       setCopyCommand({
         text: 'helm install openebs --namespace openebs openebs/openebs --create-namespace',
-        status: 'Copy to clipboard',
+        status: 'Copied to clipboard',
```
Comment (Pallavi-PH) [inline website/src/pages/Home/index.tsx:103]: Why this change?

**Review by AVRahul — COMMENTED** [review]
Due to the previous PR rebase issues the changes were missing that is why it is repeated now.

**Review by Pallavi-PH — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]
